community/cypher/acceptance/src/test/scala/org/neo4j/internal/cypher/acceptance/MatchAcceptanceTest.scala

```diff
@@ -183,22 +183,22 @@ class MatchAcceptanceTest extends ExecutionEngineFunSuite with QueryStatisticsTe
     // GIVEN
     val a = createNode()
     val b = createNode()
-    val rel1 = relate(a, b, "T")
-    val rel2 = relate(a, b, "T")
+    val rel1 = relate(a, b, "T", Map("name" -> "rel1"))
+    val rel2 = relate(a, b, "T", Map("name" -> "rel2"))
 
     val query = """
                   |MATCH (a)-[t:T]->(b)
                   |DELETE t
-                  |MERGE (a)-[t2:T]->(b)
-                  |RETURN t2
+                  |MERGE (a)-[t2:T {name:'rel3'}]->(b)
+                  |RETURN t2.name
                 """.stripMargin
 
     // WHEN
-    val result = executeWithRulePlanner(query)
+    val result = updateWithBothPlanners(query)
```

Comment:
We should move this test case to `Merge*AcceptanceTest` (node or relationship version).